schnorr_fun/tests/share_backup.rs

```diff
@@ -6,7 +6,10 @@ use secp256kfun::{marker::Secret, poly, s, Scalar};
 #[test]
 fn short_backup() {
     let secret_poly = vec![s!(6), s!(1), s!(9)];
-    let polynomial: Vec<_> = poly::scalar::to_point_poly(&secret_poly); // some polynomial coefficients
+    let polynomial: Vec<_> = poly::scalar::to_point_poly(&secret_poly)
+        .into_iter()
+        .map(|c| c.mark_zero())
+        .collect(); // some polynomial coefficients
```

Comment:
why do we need all this map stuff now?

(aside: I think `// some polynomial coefficients` is not doing much here)